defaults/main.yml

```diff
@@ -362,7 +362,15 @@ apache__combined_snippets: '{{ apache__dependent_snippets
 apache__https_enabled: '{{ ansible_local|d() and ansible_local.pki|d() and
                            (ansible_local.pki.enabled|d() | bool) and
                            apache__https_listen|length > 0 }}'
+                                                                   # ]]]
+
 
+# .. envvar:: apache__redirect_to_https [[[
+#
+# Each vhost can define a boolean ``redirect_to_https``.
+# If not set, it defaults to the value defined in ``apache__redirect_to_https``.
+# Defaults to ``True``.
```

Comment:
This is the `defaults/main.yml` and basically the last resort default, there is no more fallback default for this. You can simply remove the "Defaults to ``True``." sentence I would say.